release-notes.md

```diff
@@ -0,0 +1,24 @@
+##Release Notes
+***
+* Date: 9 June, 2015
+* Product: Z-Manager Installer
+* Description: Initial release
+
+***
+* Date: 23 July, 2015
+* Product: Z-Manager Installer
```

Comment:
May be better to drop "Z-Manager" prefix, as it is a Z-Manager repo. Just Installer or Multitenancy should be enough